app.js

```diff
@@ -1,18 +1,22 @@
 const express = require("express");
 const app = express();
-const { getCategories, getAPI } = require("./controllers");
+const { getCategories, getAPI, getReview } = require("./controllers");
 
 app.get("/api/categories", getCategories);
 
 app.get("/api", getAPI);
 
+app.get("/api/reviews/:review_id", getReview);
+
 app.use((req, res, next) => {
 	res.status(404).send({ msg: "Invalid URL" });
 });
 
 app.use((err, req, res, next) => {
 	if (err.status && err.msg) {
 		res.status(err.status).send({ msg: err.msg });
+	} else if (err.code === "22P02") {
+		res.status(400).send({ msg: "Invalid review_id" });
```

Comment:
Optional - but you may look to split out these errors into an error handling file soon :)